### PR TITLE
Fixed quality of life bugs

### PR DESCRIPTION
**Pull request by HannesTuhkala**

### CODE REVIEW

File: libplen/src/gamestate.rs [hunk 1]
```diff
@@ -257,25 +262,31 @@ impl GameState {
...
         let hit_radius = PLANE_SIZE * 2;
 
         for p1 in &self.players {
             for p2 in &self.players {
                 let distance = (p1.position - p2.position).norm();
-                if p1.id != p2.id && distance < hit_radius as f32 &&
-                    !collided_players.contains(&p1.id) && !p1.invincibility_is_on() {
-                    collided_players.push(p1.id);
+                if p1.id != p2.id && distance < hit_radius as f32 {
+                    //&& !collided_players.contains(&p1.id) {
```
Comment (Emiluren) [inline libplen/src/gamestate.rs]: Outcommented code. Pls remove

**Review by Emiluren — CHANGES_REQUESTED** [review]

File: libplen/src/gamestate.rs (hunk 1 above)
Comment (HannesTuhkala) [reply libplen/src/gamestate.rs]: It's now removed

File: libplen/src/constants.rs [hunk 2]
```diff
@@ -76,7 +76,7 @@ pub const HIT_SHAKE_MAGNITUDE: f32 = 7.;
 
 pub const KILLFEED_DURATION: f32 = 4.;
 
-pub const HURRICANE_PROBABILITY: f32 = 0.05;
+pub const HURRICANE_PROBABILITY: f32 = 0.005;
```
Comment (malcx95) [inline libplen/src/constants.rs:79]: I think this reduction is too much

**Review by Emiluren — APPROVED** [review]

**Comment by TheZoq2** [comment]
image
:thinking:

**Review by TheZoq2 — APPROVED** [review]

**Pull request merged**